### DISCUSSION TITLE
Access problem "Bad Gateway".

### DISCUSSION THREAD

**Discussion by MoallaMed** [Budibase/budibase#5745] [Help]
![bdbs](https://user-images.githubusercontent.com/102292139/167110913-422393c6-002f-44ea-a615-f0bd0b80a6c5.png)
Hello I can't access the application after the installation with kubernetes, I tried with minikube tunnel and port fordwarding but still the same error " bad gateway" .
Thank you in advance for your help

**Comment by therealarthur** [comment]
Any chance you found the solution for this problem? Im running into something similar

**Reply by AiCaptain** [reply]
Did you try a port forward to port 10000 on the proxy-service service? If it works, maybe there is a problem with your ingress configuration.
I think a port forward to the ingress controller (see screenshot of MoallaMed) did not give you access to the budibase app.
